src/components/TreeMenu/index.js

```diff
@@ -9,7 +9,7 @@ import SimpleTreeMenu from 'react-simple-tree-menu';
 import TreeMenuItem from "./TreeMenuItem";
 
 const TreeMenu = ({
-	data, searchOptions, ...props
+	data, searchOptions, hasSearch, ...props
```

Comment:
I took `hasSearch` parameter out of the props because it's only used in this file

Reply:
@aringocode Not sure about this action, `hasSearch` is a part of the `TreeMenuItem` and it can be eventually present in the Treemenu. Have you tested it with the `hasSearch={true}` option?